### ISSUE TITLE
ZIP Manager 0.1.1: Failed — blob resource

### ISSUE THREAD

**Issue by grahamperrin** [lunu-bounir/zip-manager#1]
For a `.zip` file that comprises a single `.mp4`: 

![2018-06-29 15 19 16 library](https://user-images.githubusercontent.com/192271/42097459-14cce816-7bb0-11e8-9dea-91acf6b9ff14.png)

## Environment

[Waterfox](http://www.waterfoxproject.org/) 56.2.1 on FreeBSD-CURRENT, 

```text
$ date ; uname -v
Fri 29 Jun 2018 11:13:20 BST
FreeBSD 12.0-CURRENT #5 r335024: Wed Jun 13 11:04:24 BST 2018     root@momh167-gjp4-hpelitebook8570p-freebsd:/usr/obj/usr/src/amd64.amd64/sys/GENERIC 
$ pkg info waterfox | grep Version
Version        : 56.2.1.19_2
```

**Comment by lunu-bounir** [comment]
Can you sure a page to reproduce the bug?

**Comment by grahamperrin** [comment]
Try https://github.com/PaulKinlan/videocut/files/2088018/2018-06-10.16.45.mp4.zip

I reproduced the problem with the extension added to Firefox 61.0.1 on FreeBSD-CURRENT.

**Comment by lunu-bounir** [comment]
Thanks for the report. It should be fixed on the 0.1.3 release.

**Comment by grahamperrin** [comment]
Fix confirmed. Thanks!
